khard/khard.py

```diff
@@ -320,13 +320,13 @@ def list_contacts(vcard_list, fields=[], parsable=False):
         table_header = ["uid", "name", "address_book"]
 
     if fields:
-        table_header = [x.lower() for x in fields]
+        table_header = [x.lower().replace(' ','_') for x in fields]
 
     abook_collection = AddressBookCollection('short uids collection',
                                              selected_address_books)
 
     if not parsable:
-        table.append(table_header)
+        table.append([x.title().replace('_', ' ') for x in table_header])
```

Comment:
Does it make sense to replace "_" with " " after `title()`? Isn't it the same as `capitalize()` then? I suggest to eiter use `capitalize()` or replace before using `title()`.

Reply:
I didn't test in every python version, but I found

```
>>> "python_khard".title()
'Python_Khard'
>>> "python khard".title()
'Python Khard'
```

Does this behave differently in different python versions?

Reply:
Mis-asumption on my side, sorry :(